osbs/api.py

```diff
@@ -300,7 +329,16 @@ def _create_build_config_and_build(self, build_request):
         logger.debug('build config to be named "%s"', build_config_name)
         existing_bc = self._get_existing_build_config(build_json)
 
-        if existing_bc is not None:
+        image_stream, existing_ist, image_stream_tag_name = \
```

Comment:
that's because it isn't :)
Modifying `_get_image_stream_info_for_build_request` to not return this value anymore.